SecurityPkg/Include/Library/Hash2CryptoLib.h

```diff
@@ -0,0 +1,30 @@
+/** @file
+  This library abstracts installation of the Hash 2 Service Binding protocol.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _HASH2_CRYPTO_LIB_H_
+#define _HASH2_CRYPTO_LIB_H_

```

Comment:
nit: not standard style in these headers

Reply:
This was submitted earlier on, before forming opinions about changing these things. Will update.